components/dialog/stories/template.js

```diff
@@ -75,11 +95,29 @@ export const DialogGroup = Variants({
 	testData: [
 		{
 			showModal: false,
+			isDismissible: true,
+		},
+		{
+			testHeading: "Not dismissible",
+			isDismissible: false,
+			showModal: false,
 		},
 		{
-			testHeading: "Not dismissable",
-			isDismissable: false,
+			testHeading: "With hero",
+			hasHeroImage: true,
+			isDismissible: true,
+			hasDivider: false,
 			showModal: false,
 		},
+		{
+			testHeading: "Fullscreen",
+			layout: "fullscreen",
+			showModal: false,
+		},
+		{
+			testHeading: "Takeover",
+			layout: "fullscreenTakeover",
+			showModal: false,
+		}
```

Comment:
I noticed that the "Sizing" template sizes are not rendering in the Chromatic only template. This looks like it's because `showModal: false` needs to also be set where `Sizes()` is called in the `Variants()` template (in utilities.js).
![Screenshot 2024-07-18 at 5 03 10 PM](https://github.com/user-attachments/assets/791c15e0-a745-4fd0-9160-d3afe958b620)

I'm not sure about the best way to pass that through. @castastrophe Could we add a `sizingArgs = {}` parameter to `Variants()`, that's spread after args in Sizes(), like:
`${Sizes({ Template, direction: sizeDirection, ...args, ...sizingArgs }, context)}`

Or is there another way to pass this through for just Chromatic? The only other way I could think of feels a little less clean:
```
Default.decorators = [
	// Make sure Sizes template used within Variants template changes this arg.
	(story, context) => window.isChromatic() ? story({ ...{ args: { showModal: false }}, ...context }) : story(context),
];
```

Reply:
How's this looking? Thanks for figuring it out!

https://github.com/adobe/spectrum-css/pull/2833/commits/4b535e9ff35ee5e5972c27d5e968f615016e8b0c